app/src/main/java/org/covidwatch/android/ui/util/ShareApp.kt

```diff
@@ -0,0 +1,14 @@
+package org.covidwatch.android.ui.util
+
+import android.content.Context
+import android.content.Intent
+import org.covidwatch.android.R
+
+fun Context.shareApp() {
```

Comment:
Could you please move this flie into `org.covidwatch.android.extension` package and rename it as `Context`?